site/content/en/latest/tasks/operations/customize-envoyproxy.md

```diff
@@ -822,7 +822,77 @@ spec:
 
 After applying the configuration, you will see the change in both containers in the `envoyproxy` deployment.
 
+## Customize Filter Order
+
+Under the hood, Envoy Gateway uses a series of [Envoy HTTP filters](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/http_filters)
+to process HTTP requests and responses, and to apply various policies.
+
+By default, Envoy Gateway applies the following filters in the order shown:
+* envoy.filters.http.fault
+* envoy.filters.http.cors
+* envoy.filters.http.ext_authz
+* envoy.filters.http.basic_authn
+* envoy.filters.http.oauth2
+* envoy.filters.http.jwt_authn
+* envoy.filters.http.ext_proc
+* envoy.filters.http.wasm
+* envoy.filters.http.rbac
+* envoy.filters.http.local_ratelimit
+* envoy.filters.http.ratelimit
+* envoy.filters.http.router
+
+The default order in which these filters are applied is opinionated and may not suit all use cases. 
```

Comment:
Can raise an example to explain in what case people need to reorder the filter?

Reply:
+1, a good example is ratelimit before ext auth